dist/images/Dockerfile.dpdk1911

```diff
@@ -15,21 +13,14 @@ ENV OVN_DIR=/usr/src/ovn
 ENV PATH=${PATH}:/usr/share/openvswitch/scripts
 ENV PATH=${PATH}:/usr/share/ovn/scripts/
 
-RUN dnf install -y --setopt=install_weak_deps=False \
-  make numactl-devel dpdk-devel meson \
-  unbound nc iptables ipset hostname \
-# NOTE: Fedora 32 comes with gcc version 10. This caused LD linker issues during DPDK build.
-# Install gcc version 9 and its dependencies.
-  https://kojipkgs.fedoraproject.org/packages/gcc/9.2.1/1.fc32.3/x86_64/cpp-9.2.1-1.fc32.3.x86_64.rpm \
-  https://kojipkgs.fedoraproject.org/packages/gcc/9.2.1/1.fc32.3/x86_64/libgomp-9.2.1-1.fc32.3.x86_64.rpm \
-  https://kojipkgs.fedoraproject.org/packages/gcc/9.2.1/1.fc32.3/x86_64/gcc-9.2.1-1.fc32.3.x86_64.rpm && \
-  dnf clean all && \
+RUN dnf install -y --enablerepo=PowerTools \
+  gcc make numactl-devel dpdk-devel meson \
+  unbound nc iptables ipset hostname && \
 # Install DPDK
   cd /usr/src/ && \
   curl http://fast.dpdk.org/rel/dpdk-${DPDK_VERSION}.tar.gz | tar xz && \
-  cd ${DPDK_DIR} && \
+  cd dpdk-stable-${DPDK_VERSION} && \
   meson builddir && cd builddir && \
-  meson configure -Dapps='pdump proc-info' -Dexamples='' -Dtests=false -Denable_kmods=false -Denable_docs=false && \
```

Comment:
@emmakenny , we'll need to add this line back in. My fault this got dropped when moving to centos base.

Reply:
Done